gradle/libs.versions.toml

```diff
@@ -23,7 +23,6 @@ autoService-annotations = "com.google.auto.service:auto-service-annotations:1.1.
 autoService-ksp = "dev.zacsweers.autoservice:auto-service-ksp:1.2.0"
 junit = "junit:junit:4.13.2"
 kotlin-metadata = { module = "org.jetbrains.kotlin:kotlin-metadata-jvm", version.ref = "kotlin" }
-ktfmt = { module = "com.facebook:ktfmt", version.ref = "ktfmt" }
```

Comment:
Can you restore this? Curious why it was removed as we use this for formatting

Reply:
Certainly. I was curious too, because my IntelliJ IDEA tells me that it's unused, and removing it doesn't cause `./gradlew check` to fail.

I think it's because `ktfmt` under `[versions]` is used but `ktfmt` in `[libraries]` isn't, but I'm happy to be corrected on this.

Anyway, seeing your other comment, I've reverted the change locally and will force-push soon.